src/index.html

```diff
@@ -394,6 +395,18 @@ <h2 class="font-weight-bold">Contact</h2>
                 </div>
             </form>
         </section>
+
+        <section class="col-12 py-5" id="board">
+            <div class="text-center text-black mb-5">
+                <h2 class="font-weight-bold">Board</h2>
+                <ul style="list-style-type: none">
+                    <li> President: Elsbeth Magilton</li>
+                    <li> Vice President: Jenna Vitosh</li>
+                    <li> Treasurer: Bonita Sharif</li>
+                    <li> At-Large Members:
+                         Danielle Miller • Justin Stark • Lana Zumbrunn • Ron Gallagher • Ryan Olsen • Ryan Wolff • Emma Hulsey • Jessie Hedrick
+            </div>
```

Comment:
Great job! Just a couple more suggestions on this section:
- Move above contact section
- Add closing section tag
- Because the list items are making the names off-center, change the list items to be `<p>` tags instead (and remove `<ul>` tags)
- For the div, add classes to increasing the margins on the sides, `<div class="text-center col-lg-7 col-xs-8 mx-auto text-black mb-3">` should work
- Bold board titles by using `<strong>`, for example, `<p> <strong>President:</strong> Elsbeth Magilton</p>`
- Instead of using the circles, you can use `&bull;`

Reply:
You can reach out to me over teams if you have any questions over those!